toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/MapView.kt

```diff
@@ -124,12 +124,12 @@ public fun MapView(
     locationDisplay: LocationDisplay = rememberLocationDisplay(),
     geometryEditor: GeometryEditor? = null,
     mapViewProxy: MapViewProxy? = null,
-    mapViewInteractionOptions: MapViewInteractionOptions = MapViewInteractionOptions(),
-    viewLabelProperties: ViewLabelProperties = ViewLabelProperties(),
-    selectionProperties: SelectionProperties = SelectionProperties(),
-    insets: PaddingValues = PaddingValues(),
+    mapViewInteractionOptions: MapViewInteractionOptions = remember { MapViewInteractionOptions() },
+    viewLabelProperties: ViewLabelProperties = remember { ViewLabelProperties() },
+    selectionProperties: SelectionProperties = remember { SelectionProperties() },
+    insets: PaddingValues = remember { PaddingValues() },
```

Comment:
Since `PaddingValues` is immutable, we can define a global static property for it, instead of using remember, i.e.:
```suggestion
    insets: PaddingValues = MapViewDefaults.NoPadding,
```
and elsewhere in MapView.kt:
```kotlin
object MapViewDefaults {
    public val NoPadding = PaddingValues()
   ...
}
```

See [guidelines](https://github.com/androidx/androidx/blob/androidx-main/compose/docs/compose-component-api-guidelines.md#default-expressions) for default expressions.